### DISCUSSION TITLE
hcxdumptool and Android

### DISCUSSION THREAD

**Discussion by ZerBea** [ZerBea/hcxdumptool#434] [General]
Android is not supported, but it is theoretically possible to compile hcxdumptool for Android.
Providing Android build files goes far beyond my intensions.

If you would like to compile hcxdumptool for Android, a good starting point is here:
https://developer.android.com/ndk/guides

But anyway, If you have a guide how to compile hcxdumptool for Android, feel free to share this information here.

**Comment by painowest** [comment]
awesome tool.
i use it in termux
